pyirf/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pathlib
+
+import pytest
+from gammapy.irf import load_irf_dict_from_file
+
+PROD5_IRF_PATH = pathlib.Path(__file__).parent / "resources/irfs/"
+
+
+@pytest.fixture(scope="session")
+def prod5_irfs():
+    irfs = [
+        load_irf_dict_from_file(irf_file)
+        for irf_file in PROD5_IRF_PATH.glob("*.fits.gz")
+    ]
+
+    assert len(irfs) == 3
```

Comment:
Why not run the actual download here? Checking if the files already exist?

Reply:
AT least provide an error like "You need to download using `python download_prod5_irfs.py`"

Reply:
I'd settle with an error, silently downloading stuff when the user runs local tests when one command suffices to get the data seems a bit "invasive" from my point of view.